src/simply-rets-openhouses.php

```diff
@@ -24,8 +24,13 @@ public static function getOpenHousesByListingId($listing_id) {
         ]);
 
         $response = SimplyRetsApiHelper::makeApiRequest($params, "openhouses");
+        $data = $response["response"];
 
-        return $response["response"];
+        if (!is_array($data) && property_exists($data, "error")) {
```

Comment:
For good measure, return an empty array when we get an API error.

We shouldn't even get this far for non-Enterprise users, but just in case :)